source/synthDrivers/sapi4.py

```diff
@@ -1,18 +1,19 @@
 # A part of NonVisual Desktop Access (NVDA)
```

Comment:
since we're deprecating this module can we log a deprecation warning when importing from it